packages/block-editor/src/components/block-list/style.scss

```diff
@@ -169,6 +169,9 @@
 			box-shadow: 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
 			border-radius: $radius-block-ui - $border-width; // Border is outset, so subtract the width to achieve correct radius.
 
+			// This renders the border on the GPU which fixes a Safari flicker issue.
+			will-change: transform;
```

Comment:
What transform is this referring to?

Reply:
No transform. It's meant to tell the GPU to render this element on the GPU, and I chose this instead of something like `transform: translateZ(0);` which I would consider [even more of a hack](https://stackoverflow.com/questions/34796674/what-is-advantage-of-using-will-change-property-instead-of-translate3d-for-hardw).